x-pack/metricbeat/module/azure/_meta/docs.asciidoc

```diff
@@ -97,6 +97,10 @@ so the `period` for `container_service` metricset  should be `300s` or multiples
 This metricset will collect relevant metrics from specified database accounts, these metrics will have a timegrain every 5 minutes,
 so the `period` for `database_account` metricset  should be `300s` or multiples of `300s`.
 
+[float]
+=== `app_insights`
+This metricset will collect application insights metrics, these metrics will have a timegrain every 5 minutes,
```

Comment:
Nit: s/timegrain every/timegrain of every/

Reply:
@ycombinator , thanks, the metricset does not require a timegrain of every 5 min as in the other cases so I have rewritten the text, no validation required.